www/store/features/user/userSlice.js

```diff
@@ -142,5 +142,25 @@ export const userLogout = () => {
   };
 };
 
+export const getInbox = ( email ) => {
+  return async function InboxGet( dispatch, getState ) {
+    const { data } = await client.get(
+      "participant/getmeetings",
+      {
+        params: {
+          email: email
+        }
+      }
+    );
+
+    dispatch(
+      {
+        type: "user/getmeetings",
+        payload: { userMeetings: data }
+      }
+    );
+  };
+};
```

Comment:
Not sure if this works or not but we can figure that out later. Definitely need to add the handling of the `payload` to the reducer though.

Reply:
lol i forgot I even did this, probably doesnt work.